### PR TITLE
Docs: fix footer bug

### PR DESCRIPTION
**Pull request by AndreyKuznetsovQS**

### CODE REVIEW

**Review by convoliution — CHANGES_REQUESTED** [review]
I see this fixes the bug for long pages, but now the footer is in the middle of the window on short pages:
Screen Shot 2020-03-19 at 12 57 30 PM

**Comment by AndreyKuznetsovQS** [comment]
> I see this fixes the bug for long pages, but now the footer is in the middle of the window on short pages:
> Screen Shot 2020-03-19 at 12 57 30 PM

I have tried to fix it. @convoliution please check it.

**Comment by convoliution** [comment]
It's still there 😧  (I'm certain I'm on the correct latest commit)
Screen Shot 2020-03-20 at 10 18 56 AM

File: client/verta/docs/_static/custom.css [hunk 1]
```diff
@@ -71,9 +71,9 @@ html {
 body {
   display: flex;
   flex-direction: column;
-  height: 100%;
   margin: 0;
   background: #f5f7f9;
+  min-height: 100%;
```
Comment (convoliution) [inline client/verta/docs/_static/custom.css]: This solves the problem for me in Firefox & Safari,
though I'm not sure what the best practice is regarding `vh` vs `%`.
```suggestion
  min-height: 100vh;
```

**Comment by AndreyKuznetsovQS** [comment]
> It's still there 😧 (I'm certain I'm on the correct latest commit)
> Screen Shot 2020-03-20 at 10 18 56 AM

@convoliution I added your suggestion and some changes. Please check it on your machine.

**Review by convoliution — APPROVED** [review]
I have verified this works in both Firefox and Safari. I will go ahead and merge this. Thank you Andrey 😭

**Pull request merged**